### ISSUE TITLE
New Task: Link ATN metadata to original OTN project

### ISSUE THREAD

**Issue by laurabrenskelle** [ioos/marine_life_data_network#78]
### Who is requesting this?

@ioos/marine-life

### What is being requested?

The original Trello card says: "3.6.2 Develop metadata for ATN portal to link to original OTN project, for access to additional information, such as raw detection files & receiver locations

Projects exposed in the existing ATN Data Catalog allowing users to discover acoustic projects by name, PI, species, location and view project-level metadata."

### What is the current status quo (i.e., what happens if this does not get done)?

Users accessing data via the ATN portal may not be aware of the additional information available on the OTN project page.

### Provide a description or any other important information.

This is from a Marine Life Data Network Trello card with the dates May 1, 2023 - July 31, 2023.

**Comment by MathewBiddle** [comment]
dependent on:
* #51

**Comment by MathewBiddle** [comment]
Who is the requestor here? I don't think the marine life team are appropriate to deem this something worth our time/resources. Where did this request originally come from?

**Comment by mmckinzie** [comment]
Task originally came for old ATN workplans/statements of work.

**Comment by MathewBiddle** [comment]
Thanks. Did someone ask ATN to do this? How did the task get into the old ATN workplans/statements of work?

Also, what are the prerequisites for this? Does this inherently mean ATN would be bringing in OTN data? I think we need a little bit more clarity on what the task actually is.

**Comment by mmckinzie** [comment]
Yes, past ATN network coordinators, DAC funders and our SG asked for this. Integrating acoustic data into the DAC and supporting the acoustic telemetry community is also in ATNs implementation plan.  In one form or another it's been in every one of your yearly workplans/statements of work since Axiom took of management of the DAC. But we need to revisit whether or not this task is still appropriate and of priority. The services OTN and regional nodes provides to this community have continued to evolve since some of this tasking was originally drafted and therefore our goals/objectives and the role ATN and the DAC should/could play may need to shift as a result and need to be revised or at least revisited.   

Prerequisites would be to actually start ingesting acoustic data (not just metadata) into the DAC from OTN proper, regional acoustic nodes, possibly OBIS and in some form integrate these data and related metadata into the ATN portal/catalog. Then creating a mechanism to link ATN portal to original source.

**Comment by laurabrenskelle** [comment]
@mmckinzie Is this issue a duplicate of #71?

**Comment by mmckinzie** [comment]
@laurabrenskelle They appear to be related but not duplicates. 

#71 Appears to have three parts: 1) ensure the acoustic telemetry project, receiver and/or tagging metadata are appropriately integrated into the ADR, 2) establish a link between ADR and portal and 3) develop data products/portal layers from the harvested metadata for visualization in the portal.

While this task, #78 seems more focused on ensuring appropriate links are created or embedded into any developed data layers catalog page and within the descriptive metadata about those layers to reference the original source of the metadata, e.g., by embedding hyperlinks that link portal users back to the OTN data warehouse or possibly a node's website or portal, whatever the authoritative source is and that any developed layers were discoverable from within the portal through keyword searches or other query options.    

However, the workflow and mechanism Axiom had started developing and originally used (i.e., scraping metadata from the old OTN web portal) to pull metadata into the DAC, add to ADR on a project-by-project basis, and start to develop data layers for the portal based on subsets of these metadata is no longer viable due to upgrades to the OTN system. Better endpoints are now available (e.g., GeoServer, ERDDAP). Also, the ability of the regional nodes to generate products through OTN-developed nodebooks has expanded and could be more effectively leveraged by ATN to highlight the national perspective. Though variations in the regional node's data policies/user agreements and endpoints utilized complicate this approach.

**Comment by MathewBiddle** [comment]
Closing. Not relevant and needs evaluation as to how to integrate OTN with ATN. "the landscape has changed a lot around it in terms of what powers the Nodes have to mobilize their data and metadata"
